package.json

```diff
@@ -23,6 +23,5 @@
     "eslint-plugin-import": "2.22.1",
     "eslint-plugin-prettier": "3.1.4",
     "pino-pretty": "4.3.0"
-  },
-  "dependencies": {}
+  }
```

Comment:
Not sure why this is getting removed for me - might be result of using later version of node. Can fix before I check in.

Reply:
Not really an issue, we can push this change. We will be updating to Node 14 soon anyways.